cptv-reprocess.py

```diff
@@ -97,6 +100,13 @@ def parse_args():
         help="Specify a recording range start:end or comma seperated list of recordings to reprocess id,id2,...",
     )
 
+    parser.add_argument(
+        "-alg",
```

Comment:
In most modern command line apps (except, annoyingly those using Go's built in `flags` package), a single `-` prefix is used for single char arguments and `--` is used for long versions.

In this case, defining both (`-a`, `---algorithm`) might be best